PowerPointLabs/PowerPointLabs/Ribbon1.cs

```diff
@@ -247,6 +247,21 @@ public void HighlightTextFragmentsButtonClick(Office.IRibbonControl control)
             }
         }
 
+        public void RemoveHighlightButtonClick(Office.IRibbonControl control)

```

Comment:
all codes in Ribbon1.cs & ThisAddIn.cs should be very generic and to be used by all features.
so should not create this non-generic one.
Action framework has handled this.